packages/library-base/src/jvmMain/kotlin/io/realm/kotlin/internal/platform/SystemUtilsJvm.kt

```diff
@@ -15,5 +18,12 @@ public actual val DEVICE_MODEL: String = ""
 @Suppress("FunctionOnlyReturningConstant")
 public actual fun appFilesDirectory(): String = System.getProperty("user.dir") ?: "."
 
+public actual fun assetFileAsStream(assetFilename: String): InputStream {
+    // FIXME How to get hold of the right class loader
```

Comment:
I assume we want the classloader associated with the Realm library? So probably `Realm.javaClass.classLoader`?

But I have never had to work that close with class loaders, so not 100% sure. I was going from https://www.baeldung.com/java-classloaders